hack/test-assets.sh

```diff
@@ -26,7 +26,7 @@ pushd "${OS_ROOT}" > /dev/null
   echo "Validating checked in bindata.go is up to date..."
   if ! assetdiff=$(diff -u _output/test/assets/bindata.go pkg/assets/bindata.go) ; then
 
-    echo "$assetdiff" | head -c 1000
+    echo "$assetdiff" | head -n 10
```

Comment:
Maybe more than 10, and add for example `cut -c -250` to avoid enormously long lines.